src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -148,7 +148,7 @@ void EnPoFusen_Destroy(Actor* thisx, PlayState* play) {
 /**
  * Search for Romani's actor, beacuse it's PoFusen's job to update her actor on pop.
```

Comment:
```suggestion
 * Search for Romani's actor, beacuse it's PoFusen's job to update her actor when the balloon is popped.
```
Otherwise it starts sounding like an assembly code thing.